File: pythonclass/Chapter3/python/question12.py
user_input = int(input('Enter a five digit integer: '))
if user_input < 10000 or user_input > 99999:
    user_input = int(input('Enter a five digit integer: ')) 

first_num = user_input // 10000
second_num = (user_input // 1000) % 10
third_num = (user_input // 100)%10
fourth_num = (user_input //10)%10
fifth_num = user_input % 10

if first_num == fifth_num and second_num == fourth_num:
    print("Number is a palindrome")
else:
    print('Number is not a palindrome')
    

File: pythonclass/Chapter3/java/Question12.java
import java.util.Scanner;
public class Question12 {
	public static void main(String[] args) {
	Scanner input = new Scanner(System.in);


System.out.println("Enter a five digit number: ");
int userInput = input.nextInt();

	if (userInput < 10000 || userInput > 99999);{
	System.out.println("Enter a five digit number: ");
	userInput = input.nextInt();}

int firstNum = userInput / 10000;
int secondNum = (userInput / 1000) % 10;
int thirdNum = (userInput / 100) % 10;
int fourthNum = (userInput / 10) % 10;
int fifthNum = userInput % 10;

if (firstNum  == fifthNum && secondNum == fourthNum)
	System.out.println("Number is a palindrome");
else
	System.out.println("Number is not palindrome");
	}
}